table/server/master/src/main/java/alluxio/master/table/Table.java

```diff
@@ -96,10 +98,17 @@ private Table(Database database, UdbTable udbTable, @Nullable Table previousTabl
         }
         partitions.add(newPartition);
       }
+      LOG.debug("Get {} partitions from Table {} Database {}.",
+          partitions.size(), mName, database.getName());
     } else {
       // Use all the udb partitions
       partitions =
           udbTable.getPartitions().stream().map(Partition::new).collect(Collectors.toList());
+      LOG.debug("Get {} partitions from Table {} Database {}.",
+          partitions.size(), mName, database.getName());
+      udbTable.getPartitions().stream().forEach(udbPartition ->
+          LOG.debug("Get partition: {}, from table {} Database {}.",
```

Comment:
Similar to above, we may want to guard this with `LOG.isDebugEnabled()`.

Also, the message could be something like: 
```
LOG.debug("New table {}.{} adding UDB partition: {}", ...);
```

Reply:
Changed.